lfs/lfs.go

```diff
@@ -101,6 +101,12 @@ func ResolveDirs() {
 	var err error
 	LocalWorkingDir, LocalGitDir, err = resolveGitDir()
 	if err == nil {
+		if LocalWorkingDir, err = filepath.EvalSymlinks(LocalWorkingDir); err != nil {
+			panic(fmt.Errorf("Error trying to evalsymlink '%s': %s", LocalWorkingDir, err))
```

Comment:
A few small things: 

* There's no need to pass an error to panic. `fmt.Sprintf()` would be fine.
* You can use `%q` instead of `%s` to print a quoted string.
* "evalsymlink" in the error message should probably be split out.